.travis.yml

```diff
@@ -10,19 +10,7 @@
 language: php
 
 php:
-    - 5.4
```

Comment:
@GrahamCampbell would be good to hear you are using gush :+1: :older_man: would be rewarding, i know you work with laravel but still, would be nice to hear if some people in the l5 community are using it :open_mouth: